crates/net/discv4/src/proto.rs

```diff
@@ -209,6 +211,76 @@ pub struct Neighbours {
     pub expire: u64,
 }
 
+#[derive(Clone, Debug, Eq, PartialEq)]
```

Comment:
```suggestion
/// Passthrough newtype to [`Enr`].
///
/// We need to wrap the ENR type because of Rust's orphan rules not allowing
/// implementing a foreign trait on a foreign type.
#[derive(Clone, Debug, Eq, PartialEq)]
```

Reply:
@agemanning we're using a faster RLP impl than the one you guys have upstream, if you'd be open to it you could change your dep to our crate once we publish a version on crates - should simplify parts of the dep graph too